site/posts/scottFalhmanInterview.scroll

```diff
@@ -23,7 +23,7 @@ author https://github.com/hassamalhajaji Hassam Alhajaji
  https://pldb.com/languages/java.html Java
  https://pldb.com/languages/garbage-collection-feature.html garbage collection
 
-* The other "neat" idea -- still not "mainstream", as far as I know -- is to represent programs as the same kind of objects that the system is good at manipulating: linked lisps, in the case of Lisp.  The transformation from text to list-structured representations is trivial (that's why Lisp programs have some many parentheses), and you can run that code directly in an interpreter or compiler it on the fly into fast, efficient machine code.
+* The other "neat" idea -- still not "mainstream", as far as I know -- is to represent programs as the same kind of objects that the system is good at manipulating: linked lists, in the case of Lisp.  The transformation from text to list-structured representations is trivial (that's why Lisp programs have some many parentheses), and you can run that code directly in an interpreter or compile it on the fly into fast, efficient machine code.
```

Comment:
Hi Hassam, 

Also put a comma after "interpreter" -- this will make the sentence flow better.

Kind Regards,
Liam